plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/expression/ImplementMinMax.java

```diff
@@ -40,6 +42,13 @@
 {
     private static final Capture<Variable> INPUT = newCapture();
 
+    private final boolean isRemoteCaseSensitive;
+
+    public ImplementMinMax(boolean isRemoteCaseSensitive)
+    {
+        this.isRemoteCaseSensitive = isRemoteCaseSensitive;
```

Comment:
it's not exactly about case sensitivity. For example, by default PostgreSQL is case-sensitive, but still sorts differently than Trino, so we shouldn't push min/max on varchar.

BTW do PostgresQL min/max accept COLLATE?

Reply:
I think the complaint here is that the variable is named in a misleading way? Maybe `isCollationSensitive` would be better?

Functions don't accept explicit collations - I couldn't find any examples and the variations I tried led to syntax errors.

Reply:
> I think the complaint here is that the variable is named in a misleading way

yes